sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -64,7 +64,7 @@ and then include the direct dependency in the dependencies section without the v
 If you want to take dependency on a particular version of the library that is not present in the BOM,
 add the direct dependency to your project as follows.
 
-[//]: # ({x-version-update-start;beta_com.azure:azure-messaging-servicebus;dependency})
+[//]: # ({x-version-update-start;com.azure:azure-messaging-servicebus;dependency})
```

Comment:
I find in AMQP and EH use `current` rather than `dependency`. I may lack knowledge about how auto version update works, is there any difference between this two?

Reply:
That's a good point. AFAIK, cuurent or dependency value here influences which value is set here from the version_map that is built for this library. I am not sure when we would have varying versions for the dependency and current though. Updated it to be current just to keep it same as AMQP and EH